lang/main-fr.json

```diff
@@ -752,5 +753,12 @@
         "sendFeedback": "Envoyer votre avis",
         "terms": "Termes",
         "title": "Vidéoconférence Sécurisée, entièrement en vedette et gratuite"
+    },
+    "lonelyMeetingExperience": {
+        "button": "Inviter d'autres personnes",
+        "youAreAlone": "Vous êtes le seul participant de la réunion"
+    },
+    "helpView": {
+        "Centre d'aide"
```

Comment:
this is using incorrecct syntax, that's why the inter complained

Reply:
It was copied from the base file "main.json"
What's wrong?

Reply:
UPSI i fix it